creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
+- **Execute on First Tick** is only usable for repeating command blocks with a delay. If enabled, the command will execute as soon as the block is powered, and then will wait for the delay before running again. If disabled, the command will not run the first time until the delay has finished.
+- **Delay in Ticks** is a value that defines how long a command block waits to execute after the command block receives redstone power.
 
 ## Designing a Gameplay Loop
 
-A use case of command blocks is allowing creators to create gameplay loops within Minecraft to enrich the player's experience. A simple gameplay loop would be rewarding a player for completing an action. If the gameplay loop was to give a player an emerald for placing a block of wool in a certain location, you can break down each step needed:
-
-1. Check to see if a wool block exists at a set location.
-1. Once a player places a block in the desired location, check to see if the player has already received the reward by checking their score.
-1. If the player hasn't receive the reward, give the player an emerald.
-1. If the player received a emerald, then give them a point.
-
-### Scoreboard
+A use case of command blocks is allowing creators to create gameplay loops within Minecraft. A simple gameplay loop would be rewarding a player for completing an action. If the gameplay loop was to give a player an emerald for placing a diamond block in a certain location, you can break down each step needed:
 
-The Scoreboard is a gameplay system that exists within Minecraft that can keep track of players, objectives, and score values. The scoreboard enables a wide use of options to create unique gameplay styles. In this tutorial, you will have a single objective and value in order to track when a player has completed the task.
-
-1. In Minecraft, open up the chat dialogue box (T or Enter on Windows 10 OS)
-1. Enter the following command: `/scoreboard objectives add wool_placed dummy`.
-    1. The command is adding a brand new objective called Wool_Placed. Dummy fills the argument since scores can only be adjusted via commands.
-1. Enter the following command: `/scoreboard players set @p wool_placed 0`.
+1. Check to see if a diamond block exists at a set location.
+1. If it does, check to see if the player has already received the reward.
+1. If the player hasn't received the reward, give the player an emerald.
+1. After giving the player an emerald, mark that they have received the reward.
 
 ### Setting up the first Command Block
 
 Now that we have looked at the settings of a command block, let’s set up the first block to detect a certain type of block in a location.
 
 1. Place a command block on the ground in a location near your player character.
-1. Right-click to open up the command block GUI.
+1. Open up the command block interface.
```

Comment:
I changed this because the guide already mentions how to open this up, and right-click is platform-specific.